tomoxlending/tomoxlending.go

```diff
@@ -870,7 +870,8 @@ func (l *Lending) ProcessLiquidationData(header *types.Header, chain consensus.C
 		_, collateralPrice, err := l.GetCollateralPrices(header, chain, statedb, tradingState, lendingPair.CollateralToken, lendingPair.LendingToken)
```

Comment:
if there is any other failure of `GetCollateralPrices` `GetTokenDecimal` happening in matching engine (tomox/order_processor.go tomoxlending/order_processor.go), order will be added to `reject` and everything keep working 
All we need to fix is: ignore the token pair in liquidation job below
https://github.com/tomochain/tomochain/pull/236/files#diff-9525e4c0a957e0a3e9764896c772467dR874